### DISCUSSION TITLE
Hola(hi) i need help

### DISCUSSION THREAD

**Discussion by UNCLOK333** [Pryaxis/TShock#2803] [Q&A] [answered]
Hi, I'm a little terraria youtuber and I'm doing an event, I have a question that may be somewhat obvious but I haven't found the answer yet :(
My question is: how can I make it so that when someone leaves the server when they enter again, they have to enter the password again?
I already tried everything that occurred to me, activate this option "DisableUUIDLogin" but the only thing I got is that they have to put the password of their character, I don't want that, I want to update the password while I'm playing and that those who leave already they can't get back in anyway.
Thanks in advance, regards.

**Comment by Arthri** [comment, marked as answer]
> I don't want that, I want to update the password

What do you want to update it to? Either way, TShock currently doesn't have a mechanism to auto-update passwords in this manner

**Reply by UNCLOK333** [reply]
Hi artri, I am updating the server password with the /serverpassword command but the new password is not required for the people who were already inside at the time of entering again

**Reply by Arthri** [reply]
Sorry, I don't quite understand. Do you mean that if a person leaves and then joins the server again, they don't have to enter the server password? or do they have to enter the password they originally used?

**Reply by UNCLOK333** [reply]
Excuse my English, yes, I want to say the first thing, I want that when someone leaves my server they cannot enter again because I will have changed the password and I will not have given it to them ;)

**Reply by Arthri** [reply]
This functionality currently doesn't exist in TShock, so you will need a plugin

**Reply by QuiCM** [reply]
I'm not sure that's accurate?

If I understand correctly, what is happening is @UNCLOK333 is manually setting the password with `/serverpassword`. Users who are given the password and register an account should not be able to enter the server via user account login -- instead they should be required to enter the server password (which is being manually updated as required).

This would be achieved by setting the following configuration in config.json
* `"DisableUUIDLogin": true`
* `"DisableLoginBeforeJoin": true`

**Reply by UNCLOK333** [reply]
Wow, I have done this and it works perfectly, now when someone leaves the server it asks for the password again to enter, thank you very much for the help :D

**Reply by UNCLOK333** [reply]
If anyone is interested, this is the channel where I will occupy this :)
https://www.youtube.com/channel/UCfOEUCEuwm3LoIIi7zlKacA
(is in spanish)
